src/main/java/John.java

```diff
@@ -0,0 +1,161 @@
+import commands.Deadline;
+import commands.Event;
+import commands.Task;
+import commands.Todo;
+
+
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.List;
+
+public class John {
+    //Load in the data and import it into the task list
+    static List<Task> tasks;
+    static {
+        try {
+            tasks = DataManager.readData("src/main/java/data.txt");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        System.out.println("ヽ(•‿•)ノ");
+         System.out.println("-----------------------------------");
+         System.out.println("Heya, Im ya Chatbot, call me John!\nWhat can I do for you today?");
+         // Taking input from the user
```

Comment:
comment seems a bit redundant as the function name is quite clear on the intended purpose.